### ISSUE TITLE
Export bzl_library with package files

### ISSUE THREAD

**Issue by duarten** [bazelbuild/rules_pkg#564]
This allows consumers of rules_pkg to generate docs using stardoc. An example is how skylib [does it](https://github.com/bazelbuild/bazel-skylib/blob/898cd6ddffee2a29a013ae9b70411cfe0ad16268/rules/BUILD).

**Comment by aiuto** [comment]
I have a change ready to send as a PR which will provide sources for running stardoc against.
It is blocked behind the backlog of functional changes.
Still, I don't encourage people to use startdoc themselves. It will always be incomplete documentation because the documentation for common attributes comes form other places.
The docs at https://bazelbuild.github.io/rules_pkg/ will be superior to anything you can build on your own for the forseeable  future.

**Comment by duarten** [comment]
Good to know!

Just to clarify, I'm building docs for rules I'm authoring. It's just that stardoc requires the full transitive set of my deps.

**Comment by aiuto** [comment]
#567 has a fix.    "@rules_pkg//pkg:bzl_srcs" should be usable as a bzl_library.
If anyone wants to patch that in and give it a try that will be great. Otherwise I'll wait until Friday and merge it anyway.
@guw

**Comment by duarten** [comment]
Works perfectly! I still had to add 

```
"@rules_pkg//:mappings.bzl",
"@rules_pkg//:pkg.bzl",
"@rules_pkg//pkg:bzl_srcs",
```

as dependencies to my `bzl_library`.

**Comment by aiuto** [comment]
Thanks.
That is strange.  Why :mappings.bzl?   That is not actually used by
anything.  It is just a forwarder.  Same for pkg.bzl?
Do you use those instead of //pkg:mappings.bzl and //pkg:tar.bzl?  They are
going away sometime this quarter.
https://github.com/bazelbuild/rules_pkg/issues/562

**Comment by duarten** [comment]
Ah! Thanks, I was indeed using those instead of the `pkg:*` variants.

**Comment by duarten** [comment]
Closing as this is fixed in 0.7.0.
